CONTRIBUTING.md

```diff
@@ -489,3 +489,71 @@ The following branch names are used in the Thrust project:
 On the rare occasion that we cannot do work in the open, for example when developing a change specific to an
 unreleased product, these branches may exist on `gitlab` instead of `github`. By default, everything should be
 in the open on `github` unless there is a strong motivation for it to not be open.
+
+# Release Process
```

Comment:
Other stuff we do during release:

- Prepare the changelog.
- Create annotated git tag release candidates: one for the API version (aka Thrust/CUB version), and one for any associated SDKs it goes in.
- Put the release candidate tags on GitHub.
- Make a GitHub pre-release with the release candidates and changelogs.
- Email people.
- Wait some amount of time.
- If nothing terrible happens, make a release tag, and convert the GitHub pre-release to a release.